src/Concordium/Client/Config.hs

```diff
@@ -2,6 +2,7 @@
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE QuasiQuotes #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeFamilies #-}
```

Comment:
Why this?

Reply:
Adding this (or GADTs) is needed to satisfy a constraint for string interpolation.